solvebio/global_search.py

```diff
@@ -0,0 +1,158 @@
+# -*- coding: utf-8 -*-
```

Comment:
A few thoughts on this:

```
from solvebio import GlobalSearch

s = GlobalSearch()

# Add a new method for entity search similar to how filter() works?
for result in s.entity(gene="BRCA2"):
    print(result['id'])

# Get subjects with BRCA2 in some_vault
s.filter(vault="some_vault").entity(gene="BRCA2").subjects()

# Get subjects count with BRCA2 in some_vault
s.filter(vault="some_vault").entity(gene="BRCA2").subjects_count()
```

Is it possible to cast the response objects as `Object`?
What you would do is call `Object.construct_from(i)` for each logical object result (and Vault.construct_from(i) if the result is a vault) in the _response when it is set by `execute()`.